src/routes/remove.js

```diff
@@ -20,18 +20,29 @@ const removeUserContainer = async ({username, id, reply}) => {
     return;
   }
 
-  // if not found by name - try to find by project
-  const containers = allContainers.filter(
+  // if not found by name - try to find by domain.
+  const containersByUrl = allContainers.filter(c => {
+    return c.Labels['exoframe.user'] === username && c.Labels['traefik.frontend.rule'] === 'Host:' + id;
+  });
+
+  if (containersByUrl.length && containersByUrl.length > 0) {
```

Comment:
this can just be `if (containersByUrl.length)` 🤔

Reply:
Changed :)